xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.P6257-I211.xml

```diff
@@ -0,0 +1,127 @@
+               </xi:fallback>
+            </xi:include>
+            <idno type="USEpigraphy" xml:id="NJ.Princ.PU.AM.G.P6257-I211">NJ.Princ.PU.AM.G.P6257-I211</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc>
+               <msIdentifier>
+                  <region>NJ</region>
+                  <settlement>Princ</settlement>
+                  <institution>PU</institution>
+                  <repository>AM</repository>
+                  <idno>P6257-I211</idno>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="grc" otherLangs=""/>
+                  <msItem>
+                     <p>Fragment of a Greek inscription</p>
+                  </msItem>
+               </msContents>
+               
+               <physDesc>
+                  <objectDesc ana="#fragment">
+                     <supportDesc ana="#stone.limestone">
+                        <support>
+                           <p>limestone fragment</p>
+                           <dimensions type="surface" unit="cm">
+                              <height>1.0</height>
+                              <width>1.2</width>
+                              <depth>0.9</depth>
+                           </dimensions>
+                        </support>
+                        <condition ana="#fragment.single">
+                           <p/>
+                        </condition>
+                     </supportDesc>
+                     <!--  *************************************  <layoutDesc>  ********************************* -->
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="2">
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#impressed.inscribed"> </handNote>
+                  </handDesc>
+                  
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p/>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               <history>
+                  <summary>
+                     <rs/>  <!-- content can be "fake" or blank -->
+                  </summary>
+                  
+                  <origin>
+                     <date notBefore="0001" notAfter="0200">first-second century CE</date>
+                     <placeName ref="asia.asia_minor">Turkey</placeName>
+                  </origin>
+                  <provenance type="transfer" subtype="given">
```

Comment:
Note that we're doing something different for acquisition than what the button in author mode gives; I'll write up some further documentation and I can send it along. For now, though, don't worry about including it out of author mode.

Reply:
This is duplicating the previous, and I've already corrected it to the way we're handling this material currently. Feel free to make note of how it looks here!